src/pages/home/ReportScreen.js

```diff
@@ -204,72 +207,83 @@ class ReportScreen extends React.Component {
-                                    isDrawerOpen={this.props.isDrawerOpen}
-                                />
-                            )}
-                        <ReportFooter
-                            addWorkspaceRoomErrors={addWorkspaceRoomErrors}
-                            addWorkspaceRoomPendingAction={addWorkspaceRoomPendingAction}
-                            isOffline={this.props.network.isOffline}
-                            reportActions={this.props.reportActions}
-                            report={this.props.report}
-                            isComposerFullSize={this.props.isComposerFullSize}
-                            onSubmitComment={this.onSubmitComment}
-                        />
-                    </View>
-                </FullPageNotFoundView>
-            </ScreenWrapper>
+                        <OfflineWithFeedback
+                            pendingAction={addWorkspaceRoomPendingAction}
+                            errors={addWorkspaceRoomErrors}
+                            errorRowStyles={styles.dNone}
+                        >
+                            <HeaderView
+                                reportID={reportID}
+                                onNavigationMenuButtonClicked={() => Navigation.navigate(ROUTES.HOME)}
+                                personalDetails={this.props.personalDetails}
+                                report={this.props.report}
+                                policies={this.props.policies}
+                            />
+                        </OfflineWithFeedback>
+                        {this.props.accountManagerReportID && ReportUtils.isConciergeChatReport(this.props.report) && this.state.isBannerVisible && (
+                            <Banner
+                                containerStyles={[styles.mh4, styles.mt4, styles.p4, styles.bgDark]}
+                                textStyles={[styles.colorReversed]}
+                                text={this.props.translate('reportActionsView.chatWithAccountManager')}
+                                onClose={this.dismissBanner}
+                                onPress={this.chatWithAccountManager}
+                                shouldShowCloseButton
+                            />
+                        )}
+                        <View
+                            nativeID={CONST.REPORT.DROP_NATIVE_ID}
+                            style={[styles.flex1, styles.justifyContentEnd, styles.overflowHidden]}
+                            onLayout={event => this.setState({skeletonViewContainerHeight: event.nativeEvent.layout.height})}
+                        >
+                            {/* The isTransitioning variable here is necessary so that when we are retrieving the next report data
+                            from Onyx the ReportActionsView will remount completely */}
+                            {(this.shouldShowLoader() || isTransitioning)
```

Comment:
Ah, I kind of like `isTransitioning` because it describes the state where we have a reportID in the route that doesn't match the one provided by Onyx. But maybe we don't need `isReportLoaded`.